src/commands/moderation/message/Embed.ts

```diff
@@ -150,10 +165,21 @@ export default class EmbedCommand extends BaseCommand {
 					return;
 				}
 
-				const embed = new EmbedBuilder()
+				let embed = new EmbedBuilder()
 					.setTitle(embedTitle)
 					.setDescription(embedDescription);
 
+				if (embedColour && /^#?[\da-f]{6}$/i.test(embedColour)) try {
```

Comment:
It's already a hex string, no need to have them include a hashtag.

Reply:
The hashtag is already optional in the regex test, tested